scraper.rb

```diff
@@ -36,17 +37,18 @@ def remove_prefixes(name)
 def scrape_area(url)
   noko = noko_for(url)
   area_id, area = noko.css('h2').text.tidy.match(/Constituency (\d+), (.*)/).captures
-  noko.css('.article-content tr').each do |tr|
-    tds = tr.css('td') 
-    next unless tds[0] && tds[0].text.match(/\d/)
-    pref, name = remove_prefixes(tds[1].text.tidy.sub('Col.X','Col X'))
+  area = area.match(/[a-zA-Z ]+/).to_s
```

Comment:
This is a rather odd thing to be doing. _If_ you really need this, then it should ideally be encapsulated behind something well named to explain it, and shouldn't be self-clobbering. But as I suspect you're doing this because there are **two** different "h2" elements on the page, and you're getting the text from both of them,  you really just want to make the selector on the prior line `h2.contentheading` instead.